src/elm/Session/LoggedIn.elm

```diff
@@ -274,12 +288,12 @@ viewFeedback status message =
                     " bg-red"
     in
     div
-        [ class <| "w-full sticky z-10 top-0 transition duration-500 ease-in-out bg-blue-500 hover:bg-red-500 transform hover:-translate-y-1 hover:scale-110" ++ color
+        [ class <| "w-full sticky z-10 top-0 bg-blue-500 hover:bg-red-500" ++ color
         , style "display" "grid"
         , style "grid-template" "\". text x\" 100% / 10% 80% 10%"
         ]
         [ span
-            [ class "flex justify-center items-center text-sm h-10 leading-snug text-white font-bold"
+            [ class "flex justify-center items-center transition duration-500 ease-in-out text-sm h-10 leading-snug text-white font-bold transform hover:-translate-y-1 hover:scale-110"
```

Comment:
This fixes the ability to use the `×` icon to close the message. With transitions in the parent block, it was disappearing on hover.

Reply:
😅